apps/main/src/dao/entities/locker-vecrv-info.ts

```diff
@@ -0,0 +1,25 @@
+import lib from '@/dao/lib/curvejs'
+import type { ChainId, CurveApi } from '@/dao/types/dao.types'
+import { requireLib } from '@ui-kit/features/connect-wallet'
+import type { ChainParams, ChainQuery, UserQuery, UserParams } from '@ui-kit/lib/model/query'
+import { queryFactory } from '@ui-kit/lib/model/query'
+import { userValidationGroup } from '@ui-kit/lib/model/query/user-validation'
+import { createValidationSuite } from '@ui-kit/lib/validation'
+import { curvejsValidationGroup } from './validation/curvejs-validation'
+
+async function _fetchLockerVecrvInfo({ userAddress }: ChainQuery<ChainId> & UserQuery) {
+  const curve = requireLib<CurveApi>()
+
+  return (await lib.lockCrv.vecrvInfo(curve, userAddress)).resp
```

Comment:
Can remove `async` if
```suggestion
  return lib.lockCrv.vecrvInfo(curve, userAddress).then(x => x.resp)
```

Reply:
That's even longer :sweat_smile:

Reply:
I mean yes but I'm allergic to `await` for a one-liner 😭 
Not blocking fwiw can ignore this suggestion 🫠

Reply:
I would prefer the following but that's just a question of taste :grin: 
```suggestion
  const { resp } = await lib.lockCrv.vecrvInfo(curve, userAddress)
  return resp
```

Reply:
the latter suggestion is the most declarative to me so I'll go with that